detection-rules/attachment_malicious_onenote_commands.yml

```diff
@@ -0,0 +1,15 @@
+name: "Attachment: Malicious OneNote Commands"
+description: |
+  Scans for OneNote attachments that contain suspicious commands that may indicate malicious activity.
+references:
+  - "https://www.trustedsec.com/blog/new-attacks-old-tricks-how-onenote-malware-is-evolving/"
+type: "rule"
+severity: "medium"
+source: |
+type.inbound
+and any(attachments,(.file_extension == 'one')
+and any(beta.binexplode(.), any(.scan.strings.strings, strings.ilike(., "*WshShell*", "*ExecuteCmdAsync*", "*CreateObject*", "*Wscript.Shell*","*schtasks*", "*CreateProcess*","*winmgmts*", "*SetEnvironmentVariable*"))))
```

Comment:
```suggestion
    type.inbound
    and any(attachments, .file_extension == 'one'
        and any(beta.binexplode(.),
            any(.scan.strings.strings,
                strings.ilike(., 
                    "*WshShell*",
                    "*ExecuteCmdAsync*",
                    "*CreateObject*",
                    "*Wscript.Shell*",
                    "*schtasks*",
                    "*CreateProcess*",
                    "*winmgmts*",
                    "*SetEnvironmentVariable*",
                )
            )
        )
    )
```

Reply:
added preceding spaces to make it valid YAML, and a tweak to formatting for readability